lib/av_client/types.ts

```diff
@@ -330,3 +264,151 @@ export type ReadableOptionSelection = {
   title: string
   text?: string
 }
+
+/** New stuff */
+
+// Latest Config
+export interface LatestConfig {
+  items: LatestConfigItems
+  receipt?: string
+  affidavit?: AffidavitConfig
+  castRequestItemAttachmentEncryptionKey?: string
+}
+
+export interface LatestConfigItems {
+  thresholdConfig: ThresholdConfig;
+  voterAuthorizerConfig: VoterAuthorizer;
+  ballotConfigs: BallotConfigMap;
+  contestConfigs: ContestConfigMap;
+  // votingRoundConfigs: VotingRoundConfig[];
```

Comment:
Do we need this around?

Reply:
We're using it on the voting round branch that comes next to this one